pkg/api/routes.go

```diff
@@ -55,7 +55,7 @@ func NewRouteHandler(c *Controller) *RouteHandler {
 func (rh *RouteHandler) SetupRoutes() {
 	rh.c.Router.Use(AuthHandler(rh.c))
 
-	if !isBearerAuthEnabled(rh.c.Config) && rh.c.Config.AccessControl != nil {
+	if !isBearerAuthEnabled(rh.c.Config) && isAuthnEnabled(rh.c.Config) {
```

Comment:
After fixing isAuthnEnabled() above, 

if rh.c.Config.AccessControl != nil && isAuthnEnabled() && !isBearerAuthEnabled {
}

Reply:
Also add a log here saying access control is being enabled

Reply:
done, thank you!